engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/Atom.java

```diff
@@ -203,6 +205,25 @@ boolean hasFunctionalDispatch() {
     return true;
   }
 
+  @ExportMessage
+  boolean isDate(
+    @CachedLibrary("this") InteropLibrary iop
+  ) {
+    var dateConstructor = Context.get(iop).getDateConstructor();
+    if (dateConstructor.isPresent()) {
```

Comment:
`dateConstructor` can lead to NPE if the module is not present?

Reply:
also `dateConstructor.ifPreset(...).orElse(false)`

Reply:
`getDateConstructor()` always returns non-`null` value, but it can be `!isPresent()`.